src/vs/platform/windows/common/windows.ts

```diff
@@ -309,19 +309,36 @@ export interface IPathsToWaitFor {
 	waitMarkerFilePath: string;
 }
 
+export interface IPathData {
+
+	// the file path to open within a Code instance
+	fileUri?: UriComponents;
+
+	// the line number in the file path to open
+	lineNumber?: number;
+
+	// the column number in the file path to open
+	columnNumber?: number;
+}
+
+export interface IPathsToWaitForData {
```

Comment:
@aeschli can `IPathsToWaitFor` extend `IPathsToWaitForData` to avoid repetition of fields?

Reply:
👍